docassemble/MAVirtualCourt/data/questions/basic-questions.yml

```
@@ -275,6 +275,12 @@ fields:
   - Birthdate: children[i].birthdate
     required: False
 ---
+code: |
+  # This is a workaround for the fact that user role isn't
+  # defined in the wizard-generated file yet.
+  if not defined('user_role'):
+    user_role = False # this should get defined to "plaintiff" or "defendant" in the actual interview
```

What are the ramifications of this? Does this cause a silent failure later?

---

From description and some testing, it seems like `False` triggers a question to be asked. Maybe worth adding to comment in the future.